lib/debug/tracer.rb

```diff
@@ -95,6 +95,8 @@ def puts msg
     end
 
     def minfo tp
+      return "{block}" if tp.event == :b_call
```

Comment:
Since `block` is a valid method name, I think we should use additional symbols to distinguish that it's a Ruby block call.

```
# block call
DEBUGGER (trace/object) #th:1 #depth:5  1 is used as a parameter int of {block} at target.rb:6
# method call named block
DEBUGGER (trace/object) #th:1 #depth:4  1 is used as a parameter int of Object#block at target.rb:10
```

I also thought about `&block` but users may assume `block` to be the block argument's name?